app/Http/Controllers/City/GetCityController.php

```diff
@@ -3,13 +3,17 @@
 namespace App\Http\Controllers\City;
 
 use App\Http\ViewModels\CityViewModel;
+use App\Models\City;
+use App\Transformers\CityTransformer;
 use Illuminate\Http\JsonResponse;
 
 class GetCityController
```

Comment:
Get city suggests it get's a particular city, like an url that goes like `/api/cities/{cityId}`, if it is the get cities pagination endpoint, rename it to GetCities or ListCities

Reply:
also remember to use the transformer. passing a paginator like this will work just fine
 ` return responder()->success($cities, CityTransformer::class)->respond();`